lib/mayaUsd/utils/utilFileSystem.h

```diff
@@ -143,6 +143,21 @@ void unmarkPathAsPostponedRelative(
     const PXR_NS::SdfLayerHandle& layer,
     const std::string&            contentPath);
 
+/*! \brief Performs handling of a USD asset path attribute that maybe relative to a layer.
+Whether the file path is required to be relative is defined by the primvar which name is supplied
+through 'primVarName' variable. If the relative file path is required and the layer is saved on disk
```

Comment:
Is it not an 'optionVar' name and not a 'primVar' name?

Reply:
Yeah, you are right. I'll fix it